apps/site/components/Common/AvatarGroup/index.tsx

```diff
@@ -30,21 +36,31 @@ const AvatarGroup: FC<AvatarGroupProps> = ({
 
   return (
     <div className={styles.avatarGroup}>
-      {renderAvatars.map((avatar, index) => (
-        <Avatar
-          src={avatar.src}
-          alt={avatar.alt}
-          fallback={getAcronymFromString(avatar.alt)}
-          key={index}
-        />
+      {renderAvatars.map(({ ...avatar }) => (
+        <Fragment key={avatar.nickname}>
+          {avatar.url ? (
+            <Tooltip
```

Comment:
Let's always have the Tooltip component but with an optional argument of `disabled` so that the tree is the same, but tooltip just gets disabled.

Reply:
I solved it by adding the `pointer-events-none` class